deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
 
 ```
 kubectl delete namespace $OTEL_NAMESPACE
-```
\ No newline at end of file
+```
+
+## Try it out yourself (Optional)
+
+This section contains steps that you can follow to actually see redaction you setup earlier in action. Since there is no expected outcome from our kubernetes deployment due to a lack of telemetry data, the redaction processor setup in our collector cofiguration is not applied to anything. In this section we will introduce a source of telemetry data to actually see the redaction in action.  
+
+### Prerequisites 
+You should be able to follow all steps mentioned in this README till [Verify the Deployment](#verify-the-deployment) successfully. This ensures that there are no permissions issues and you are able to succeesfully run a GKE cluster and connect to it. 
+
+### Source of telemetry data
+
+We would be using JSON files containing telemetry data in OTLP format as the source of telemetry data. A sample file has been provided here - [testdata.json](./otlp-data/testdata.json). The collector running in the cluster would read this file and treat the data in this file as if it were coming from a running application. 
+
+### Updating the collector configuration to receive data from JSON file
+
+Next, we would update our collector configuration file - [otel-config.yaml](./otel-config.yaml) to add a reciever that is able to receive telemetry data from the added JSON file. 
+
+Under the `receivers` section in the config file, add the following configuration - 
+
+```
+  otlpjsonfile:
+    start_at: "beginning"
+    include:
+      - "/mnt/testdata/*.json"
+
+```
+For more details about this particular receiver, check [otlpjsonfilereceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/otlpjsonfilereceiver).
+
+***NOTE:** The path in `include` points to where the file would be mounted within the kubernetes cluster environment and is therefore different from where it is present on your local machine. More information on this in [the next section](#making-the-json-file-available-within-the-cluster).*
+
+Next, add this receiver in the traces pipeline, so your trace pipeline looks like - 
+```
+ traces:
+      receivers: [otlp, otlpjsonfile]
+      processors: [memory_limiter, batch, resourcedetection/gke, redaction/credit_cards]
+      exporters: [googlecloud, logging]
+```
+ 
+### Making the JSON file available within the cluster 
+
+Simply adding the JSON file within the directory does not grant the running cluster the access to this file. In order to get access to this file within the kubernetes environment, we would be mounting this file as a kubernetes `configMap` for a new deployment - 
```

Comment:
```suggestion
Simply adding the JSON file within the directory does not grant the running cluster the access to this file. In order to get access to this file within the Kubernetes environment, we will mount this file as a Kubernetes ConfigMap for a new deployment - 
```

Reply:
when it's not clear how a k8s object should be stylized (such as `configMap` vs ConfigMap), I usually fall back to the upstream documentation as a guide: https://kubernetes.io/docs/concepts/configuration/configmap/

please ctrl+f this change throughout these steps